PowerEditor/src/ScintillaComponent/FindReplaceDlg.rc

```diff
@@ -57,8 +57,8 @@ BEGIN
     CONTROL         "Re&gular expression",IDREGEXP,"Button",BS_AUTORADIOBUTTON,12,167,78,10
     CONTROL         "&. matches newline",IDREDOTMATCHNL,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,93,167,101,10
 
-    GROUPBOX        "",IDC_REPLACEINSELECTION,192,50,180,23
-    CONTROL         "In select&ion",IDC_IN_SELECTION_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,200,58,59,10
+    GROUPBOX        "",IDC_REPLACEINSELECTION,162,50,180,23
+    CONTROL         "In select&ion",IDC_IN_SELECTION_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,170,58,90,10

```

Comment:
It makes "In selection" checkbox float in the middle of Mark section...
However, "In selection" checkbox in Mark section should belong to "Mark all" & "Clear all marks" button.
Have you any idea for the solution?

Reply:
@donho 
maybe this
![Screenshot 2024-06-11 180703](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/55940305/ebd56c78-ce78-4f22-abcf-b28bcaa223fe)